docs/getting-started/getting-started-with-low-code-airflow.md

```diff
@@ -0,0 +1,165 @@
+
+## 1. Setup Prophecy Fabric for Airflow
+
+Prophecy introduces the concept of a Fabric to describe an execution environment. In this case, we create a Fabric to connect to Airflow, and create and schedule DAGs in it.
+For this guide, we would be using Prophecy Managed Airflow, so an external Airflow instance is not required.
+
+![Create Fabric](img/2-4-create-fabric.png)
+
+Setting up a Fabric is very straightforward. Click the **(1) Create Entity** button, and choose **(2) Create Fabric** option. The Fabric creation is composed of two steps: Basic Info and Providers setup.
+On the Basic Info screen, enter a **(1) Fabric Name**, **(2) Fabric Description**, and choose the **(3) Team** that’s going to own the Fabric.
+
+Once ready, click **(4) Continue**.
+
+![Fill_fabric_details.png](img/Airflow_1.2_Create_Fabric.png)
+
+Since we’re setting up a Fabric connected to Airflow, we choose **Airflow** as the **(1) Provider Type** and **Prophecy Managed** as the **(2) Provider**.
+For connecting to Prophecy Managed Airflow, you don't need to provide any other details, so go ahead and click on **(3) Continue**.
+
+![Add_connection.png](img/Airflow_1.3_Add_connection.png)
+To be able to Run your Databricks Pipelines and Models, you need to have connection from Prophecy Managed Airflow to your Databricks Environment.
+Click on **(1) Add Connection** button. This opens up the Add connection form.
+
+![Add_connection_form.png](img/Airflow_1.4_Add_connection_form.png)
+Select Databricks Spark or Databricks SQL in **(1) Connection Type**. Now under the **(2) Fabric**, you would select the already created Fabric for Databricks Spark or Databricks SQL and Prophecy would setup the connection.
+You can provide a description in the **(3) Description**.
+Once done, click **(4) Save**.
+
+![Fabric_complete.png](img/Airflow_1.5_Complete_Fabric_setup.png)
+You can add more than one connection for Databricks Spark,Databricks SQL, Email, HTTP, etc.
+Once done click **(1) Complete** to complete your Fabric Setup.
+
+You should now start seeing a new Fabric of type `Airflow` in the Fabric listing page.
+
+## 2. Create an Airflow Job
+
+A Job is an entity that contains Gems to represent a DAG consisting of various Tasks (Pipelines/Models/Scripts, etc) which you can Run once or schedule to run at a frequency. Each Job would represent an Airflow DAG in Python.
+
+Let's see how to create an Airflow Job in Prophecy.
+
+![Create_Job](img/Airflow_2.1_Create_Job.png)
+
+Click the **(1) Create Entity** button, and choose **(2) Create Job** option.
+
+On the first screen, you would provide the Basic Info of the Job.
+
+In **(1) Project**, select the Project in which you want to create the Job. You can pick the existing Databricks Spark or SQL project here where you have created Pipelines/Models.
```

Comment:
This set of sentences starting with `In ...` is challenging to read. There are two options here: 

1. Write it in proper English as a set of connected sentences. Screenshot with the example below. 
2. Or write it as a bullet list. 

Please note there should not be any uppercase after a comma. 

<img width="1062" alt="Screenshot 2023-08-23 at 5 38 08 PM" src="https://github.com/SimpleDataLabsInc/prophecy-docs/assets/3248329/1e20f585-f76d-49a4-90d6-e3a5c1b4016b">